Description: Correct pod NAME issue in LedgerSMB/Template/Elements.pm
 Has a colon instead of a dash in the pod NAME entry.
Bug: http://sourceforge.net/tracker/?func=detail&aid=3472619&group_id=175965&atid=1005191
Author: Robert James Clay <jame@rocasa.us>

--- a/LedgerSMB/Template/Elements.pm
+++ b/LedgerSMB/Template/Elements.pm
@@ -1,6 +1,6 @@
 =head1 NAME
 
-LedgerSMB::Template::Elements:  Template Utility Functions
+LedgerSMB::Template::Elements - Template Utility Functions
 
 =head1 SYNOPSIS
 
